slices/libvpx7.yaml

```diff
@@ -0,0 +1,9 @@
+package: libvpx7
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+    contents:
+      /usr/lib/*-linux-*/libvpx.so.7*:
```

Comment:
These files seem missing for `armhf`:

```
/usr/lib/arm-linux-gnueabihf/vfp/neon/libvpx.so.7
/usr/lib/arm-linux-gnueabihf/vfp/neon/libvpx.so.7.0
/usr/lib/arm-linux-gnueabihf/vfp/neon/libvpx.so.7.0.0
```

From https://packages.ubuntu.com/jammy/armhf/libvpx7/filelist.

Reply:
added